core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionNetworkOutput.java

```diff
@@ -176,6 +177,9 @@ public boolean send(Transmission transmission) {
                 if (ioe instanceof ConnectTimeoutException) {
                     transmissionPolicyManager.suspendInSeconds(TransmissionPolicy.BLOCKED_BUT_CAN_BE_PERSISTED, DEFAULT_BACKOFF_TIME_SECONDS);
                 }
+            } catch (IllegalStateException e) {
+                transmissionPolicyManager.stop(SHUTDOWN_TIME, TimeUnit.SECONDS);
```

Comment:
@Dmitry-Matveev i agree. It is better to let it try. DEFAULT_BACKOFF_TIME_SECONDS may be a bit long time. It would be a very "big" bug if the app can recover from a illegall state exception.

I had to fix the problem in 1.0.8 with transmissionPolicyManager.suspendInSeconds() because in just mater of minutes it filled like 60 gig of diskspace. It works very well. Not filling up my server:) I will make new changes.